demo/load-tests/load-with-requirejs.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>EME Encryption Scheme Polyfill - load with RequireJS</title>
+    <script src="https://unpkg.com/requirejs@^2/require.js"></script>
+    <script>
+      async function testLoader() {
+        return await new Promise((resolve, reject) => {
```

Comment:
nit: the await isn't needed since you return a Promise.  You also don't need the `async` since you already return a Promise.

Reply:
Somehow, I felt it was more consistent to use async on each one to show that it was treated as such.  But I've removed them now.